src/containers/CharacterProfile/MoveList.js

```diff
@@ -40,35 +40,33 @@ class MoveList extends Component {
     this.props.navigation.navigate('attackDetails', {move, index});
   }
 
-  renderByOrientation(orientation, moves) {
-    if (orientation == 'landscape') {
-      return (
-        <Spreadsheet moves={moves} onMovePress={(move, index) => this.onMovePress(move, index)} />
-      );
-    } else {
-      // plain list of moves
+  renderByOrientation(isPortrait, moves) {
       return (
+        <View>
+        <Spreadsheet isPortrait={isPortrait} moves={moves} onMovePress={(move, index) => this.onMovePress(move, index)} />
```

Comment:
We could just wrap a `View` component around these two different move lists and use `isPortrait` to set the hidden style or not instead of having to pass down `isPortait`.

Reply:
Is there performance issues having both the spreadsheet and regular movelist on the page at the same time, and hiding just one of them?

If not, then this `renderByOrientation()` method is no longer needed since we don't do a check to see which one we should render. In this case if you wanted to separate it for organization, create a `renderSpreadSheet()` and `renderRegularMoveList()` methods or something of that nature.

Reply:
I also +1 this. Adding these little tweaks will seriously help perf. On web, perf isn't so bad because you can hide stuff, but on mobile any issue is magnified.